examples/howto/updating_priors.myst.md

```diff
@@ -10,56 +10,61 @@ kernelspec:
   name: python3
 ---
 
-# Updating priors
+(updating-priors)=
+# Updating Priors
+
+:::{post} January, 2017
+:tags: priors
+:category: intermediate, how-to
+:::
 
 +++
 
-In this notebook, I will show how it is possible to update the priors as new data becomes available. The example is a slightly modified version of the linear regression in the [Getting started with PyMC3](https://github.com/pymc-devs/pymc3/blob/master/docs/source/notebooks/getting_started.ipynb) notebook.
+In this notebook, we will show how it is possible to update the priors as new data becomes available.
```

Comment:
I would add a note that this is actually a bit sketchy as the MC error can creep in. I think the last plot actually shows that as some posteriors jump.

Reply:
+1